### PR TITLE
Add mongo db, page for fetching new data and saving

### PR DESCRIPTION
**Pull request by johnhaggkvist**

### CODE REVIEW

**Comment by johnhaggkvist** [comment]
:+1: :)

File: store.js [hunk 1]
```diff
@@ -1,51 +1,65 @@
...
+  stores: [{
+    store: String,
+    stock: Number
+  }]
+});
 
-  var get = function (success, error) {
-    var count = 0, stores = [], _fetchPagesFrom = function (page) {
-      var url = 'http://www.systembolaget.se/api/site/findstoresincountywhereproducthasstock/Stockholms%20l%C3%A4n/896008/';
+StoreSchema.statics.getLatestFetch = function(success, error) {
```
Comment (johnhaggkvist) [inline store.js:14]: Maybe `statistics` :)

Comment (larsericsson) [inline store.js:14]: `statics` is part of [mongoose API](http://mongoosejs.com/docs/2.7.x/docs/methods-statics.html) to create static Model methods

Comment (johnhaggkvist) [inline store.js:14]: ah! :D

**Pull request merged**